tests/meltano/core/job/test_job.py

```diff
@@ -176,3 +183,13 @@ def test_fail_stale(self):
         assert job.fail_stale()
         assert job.has_error()
         assert "5 minutes" in job.payload["error"]
+
+
+def windows_signal_process(signal: int):
+    # Replace with signal.raise_signal once Python 3.7 has been dropped
+    # https://stackoverflow.com/questions/35772001/how-to-handle-a-signal-sigint-on-a-windows-os-machine
+    import ctypes
+
+    ucrtbase = ctypes.CDLL("ucrtbase")
+    c_raise = ucrtbase["raise"]
+    c_raise(signal)
```

Comment:
Overall, this looks really good. Thanks very much for creating this, @visch. 👍 

I like the comment and inline URL but what do you think of swapping this with a link to a new issue on Meltano side?

Perhaps something like "Remove/replace Windows 'raise' workaround in after Python 3.7 EOL."

And that issue can have the additional info and context?

Reply:
Done, makes a lot of sense, left Python 3.7 in there so when someone greps for 3.7 later hopefully it'll come up